proposed/extended-coding-style-guide.md

```diff
@@ -1019,6 +1019,14 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Type Casting
+
+There MUST be no spaces inside type casting parentheses:
```

Comment:
Shouldn't it be more along the lines of:
``There MUST NOT be any spaces inside the type casting parentheses:``?